modules/services/disciplina.nix

```diff
@@ -0,0 +1,81 @@
+{ config, lib, pkgs, ... }: with lib;
+
+let
+  cfg = config.services.disciplina;
+
+  attrsToFlags = set:
+    let
+      render = name: value:
+        "--" + name + (optionalString (isString value) (" " + value));
+
+      renderList = name: value:
+        if isList value
+        then map (render name) value
+        else [ (render name value) ];
+    in
+    concatStringsSep " " (concatLists (mapAttrsToList renderList set));
```

Comment:
prefer `concatMap` instead of `concatLists (map ...)` for better performance

Reply:
Sorry, no idea how to do this.